### PR TITLE
Update Aviation Cockpits config

### PR DESCRIPTION
**Pull request by AnticlockwisePropeller**
- Standardised mix of tabs and spaces to tabs only
- Added names/descriptions for existing cockpits
- Added support for Fw190, Bearcat, and Su30 cockpits

### CODE REVIEW

File: GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg [hunk 1]
```diff
@@ -86,75 +88,246 @@
...
+		amount = 1
+		maxAmount = 1
+	}
+	!MODULE[ModuleReactionWheel] {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		type = Structural
+		basemass = -1
+		volume = 20 //
```
Comment (rsparkyc) [inline GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg]: comment with no content

Comment (stratochief66) [inline GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg]: And what is this ModuleFuelTanks for, anyway? 
With RO using RealFuels, we tend to place resources as TANKS within ModuleFuelTanks
https://github.com/KSP-RO/RealismOverhaul/pull/1569/files#diff-1119e2770a2add02ea460939f55e0d7bR44

Comment (AnticlockwisePropeller) [inline GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg]: I based the new configs on the original config @NathanKell had written for the Interceptor?Cockpit. I can delete the Resources and put them as TANKS within the ModuleFuelTanks, no probs.

**Comment by AnticlockwisePropeller** [comment]
Deleted comment with no content and replaced RESOURCES with ModuleFuelTanks TANKS, as suggested.

**Comment by stratochief66** [comment]
@AnticlockwisePropeller thanks for the changes. I'd like someone to have a chance to test these in game before they are merged, and that won't be me because I can't build or fly planes :package: 
So, this PR looks good, it just may sit for a while until someone with the right skill is available to test it out.

**Comment by NathanKell** [comment]
Took a quick peek at the code. Starting line 26 it's got problems: @RESOURCE modifies the first node of that name, but then the changes inside are insertions not additions so the node will have double values. That, and any other cases of that, would need to be fixed.

Loving that this is getting more love though! :)

**Comment by AnticlockwisePropeller** [comment]
Thanks, Nathan! I'll fix that right away!

File: GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg [hunk 2]
```diff
@@ -247,3 +434,88 @@
...
+
+//Sukhoi Su30
+@PART[Mk1?Su30?Cockpit]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@title = Su-30 Cockpit
+	@manufacturer = Sukhoi
+	@description = Twin seat fighter cockpit.
+	%rescaleFactor = 1.2 // 1.5m end diameter
+
```
Comment (Olympic1) [inline GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg]: This part has messed up nodes, so add
```
	!node_stack_bottom = DEL
	!node_stack_bottom = DEL
	!node_stack_top = DEL
	node_stack_bottom = 0.0, -2.21851, 0.0, 0.0, -1.0, 0.0, 1
	node_stack_top = 0.0, 1.44135, 0.0, 0.0, 1.0, 0.0, 1
```

Comment (AnticlockwisePropeller) [inline GameData/RealismOverhaul/RO_SuggestedMods/AviationCockpits/Cockpits.cfg]: Nice catch! Didn't notice that bc the node was in the right place, just the wrong way around

**Comment by NathanKell** [comment]
Looks good to me, including Olympic's fix.

**Comment by NathanKell** [comment]
Uh, how do we apply the suggestion?

**Pull request merged**